package.json

```diff
@@ -75,40 +73,45 @@
   },
   "dependencies": {
     "attr-accept": "^1.1.3",
-    "prop-types": "^15.5.7"
+    "prop-types": "^15.6.2"
   },
   "devDependencies": {
-    "@commitlint/cli": "^3.2.0",
-    "@commitlint/config-angular": "^3.0.3",
-    "@commitlint/prompt": "^3.2.0",
-    "@commitlint/prompt-cli": "^3.2.0",
+    "@commitlint/cli": "^7.0.0",
+    "@commitlint/config-angular": "^7.0.1",
+    "@commitlint/prompt": "^7.0.0",
+    "@commitlint/prompt-cli": "^7.0.0",
     "babel-cli": "^6.9.0",
-    "babel-core": "^6.9.1",
-    "babel-eslint": "^7.1.1",
-    "babel-jest": "^21.0.0",
+    "babel-core": "^6.26.3",
+    "babel-eslint": "*",
```

Comment:
Please, prefer normal caret version.

Reply:
This is added by `install-peerdeps` so I don't want to lock it